override.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExtensionDirectories=/var/lib/extensions/tailscale
```

Comment:
cool update, only nit comment: seems like this file does not end in new line, idk if systemd respects POSIX standard, but might want to make sure files end in new line just to make sure GNU programs (and maybe systemd) has no issues reading it

Reply:
I'll take care of that later today. Thanks!